<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>opiswykres</class>
 <widget class="QMainWindow" name="opiswykres">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="sourceLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="selectBtn">
       <property name="text">
        <string>Wybierz plik</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="destLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="CopyBtn">
       <property name="text">
        <string>Kopiuj plik</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>97</width>
      <height>137</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="SaveBtn">
         <property name="text">
          <string>Zapisz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OpenBtn">
         <property name="text">
          <string>Otwórz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFont">
       <property name="text">
        <string>Czcionka</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonColor">
       <property name="text">
        <string>Kolor</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>456</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuitDescription"/>
  </widget>
  <action name="actionQuitDescription">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/close.ico</normaloff>:/icons/icons/close.ico</iconset>
   </property>
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
